exporter/loadbalancingexporter/README.md

```diff
@@ -192,9 +192,9 @@ exporters:
         # except the endpoint
         timeout: 3s
     resolver:
-      awsCloudMap:
+      aws_cloud_map:
         namespace: aws-namespace
-        serviceName: aws-otel-col-service-name
+        service_name: aws-otel-col-service-name
```

Comment:
```suggestion
        service_name: otel-col-service-name
```
Teeny tiny nit. Vendor agnostic right :)

Reply:
Should I change `aws-namespace` as well? It's pretty hard to make AWS Cloud Map resolver cloud agnostic :)